src/benchmark/config_parser.py

```diff
@@ -5,11 +5,13 @@
 
 
 class TestResultParser:
-    def get_tests_list(self, config):
+    @staticmethod
+    def get_tests_list(config):
```

Comment:
why all the changes in src/benchmark/config_parser.py are done?

Reply:
because it's the right thing to do)

Reply:
I think it's just revert of get_path_to_inference_folder and removal of our config

Reply:
please, create separate Jira task pull request for refactoring, if you see it feasible.
In current PR we need to introduce changes needed to merge to upstream and fix review comments only.